File: go/raindrops/raindrops.go
//Package raindrops does the raindrops stuffs
package raindrops

import "strconv"

//Convert gives us all the sounds
func Convert(n int) string {
	rainspeak := ""
	if n%3 == 0 {
		rainspeak += "Pling"
	}
	if n%5 == 0 {
		rainspeak += "Plang"
	}
	if n%7 == 0 {
		rainspeak += "Plong"
	}
	if len(rainspeak) == 0 {
		rainspeak = strconv.Itoa(n)
	}
	return rainspeak
}

File: ruby/raindrops/raindrops.rb
# frozen_string_literal: false

# class Raindrops
class Raindrops
  def self.convert(num)
    rainspeak = ''
    (num % 3).zero? &&  rainspeak << 'Pling'
    (num % 5).zero? &&  rainspeak << 'Plang'
    (num % 7).zero? &&  rainspeak << 'Plong'
    rainspeak.empty? && rainspeak = num.to_s
    rainspeak
  end
end
